### PR TITLE
Improve path mapping support

### PR DESCRIPTION
**Pull request by clydin**
Fixes: #11301

### CODE REVIEW

File: packages/ngtools/webpack/src/paths-plugin.ts [hunk 1]
```diff
@@ -109,6 +131,9 @@ export function resolveWithPaths(
     return;
   }
 
+  // TODO: The following is used when there is more than one potential and will not be
+  //       needed once this is turned into a full webpack resolver plugin
+
```
Comment (alan-agius4) [inline packages/ngtools/webpack/src/paths-plugin.ts]: Shouldn’t we return the dirname when package.jaon exists?

**Comment by alan-agius4** [comment]
Veru nice 😀

File: packages/ngtools/webpack/src/paths-plugin.ts (hunk 1 above)
Comment (clydin) [reply packages/ngtools/webpack/src/paths-plugin.ts]: Would you mind opening a followup PR with your suggested changes once this is merged?

Comment (alan-agius4) [reply packages/ngtools/webpack/src/paths-plugin.ts]: Sure thing, I can do that 😊

**Pull request merged**